### DISCUSSION TITLE
помогите с обходом

### DISCUSSION THREAD

**Discussion by Mihrutkin** [hufrea/byedpi#156] [General]
Все хорошо работало с командой
--disorder 1 --fake 0 --ttl 1 -A torst --tlsrec 1+s --debug 1
Ростелеком. Сегодня рубанули.
Помогите с командами. Спасибо

**Comment by Black-Semargl** [comment]
У меня с  **-F --split 2 --disorder -1+s**  заработало

**Comment by cravvler9** [comment]
> У меня с **-F --split 2 --disorder -1+s** заработало

Всё чётко, спасибо!

**Comment by tkzv** [comment]
> У меня с **-F --split 2 --disorder -1+s** заработало

Мегафон, не помогает.

**Comment by Seychik23** [comment]
> У меня с **-F --split 2 --disorder -1+s** заработало

Ростелеком (Москва), не помогает

**Comment by Norayr44444** [comment]
-F --split 1 --disorder -1
Так попробуйте 
-F --split 1 --fake -1
Или так
(Замечание: при повышении --split >1
перестает работать)

**Comment by Seychik23** [comment]
> -F --split 1 --disorder -1 Так попробуйте

Работает, спасибо

**Comment by Black-Semargl** [comment]
похоже каждые несколько дней схему надо будет менять...

**Comment by Norayr44444** [comment]
> похоже каждые несколько дней схему надо будет менять...




Ваш вариант у меня проработал около часа.. следовательно, скорее всего чаще)

**Comment by tkzv** [comment]
> -F --split 1 --disorder -1 Так попробуйте

Спасибо, пока работает.

**Comment by stormy23dev** [comment]
> -F --split 1 --disorder -1 Так попробуйте -F --split 1 --fake -1 Или так (Замечание: при повышении --split >1 перестает работать)

Оба варианта не помогают (ютуб видео не грузит, linux, ростелеком)

**Comment by D10n1x** [comment]
у меня с таким вариантом заработало 
```command: --disorder 1 --fake 0 --md5sig --ttl 1 -A torst --timeout 1 --tlsrec 3+s --debug 1 --tfo```
но есть минус поиск перестает корректно работать периодически выдавая ошибку 
```При соединении с www.google.com произошла ошибка. PR_END_OF_FILE_ERROR```

**Comment by FightingForFun** [comment]
tlsrec самостоятельно работает или он обязательно с --fake должен быть? ... чет не пойму.

+n смещение имеет смысл? а то в код глянул, он никаких флагов не добавляет. 

а то печатаю, а то ли печатаю)

**Reply by slesru** [reply]
работает самостоятельно.

только вот у меня ничо щас не работает, совсем.
эти уроды забанили ещё и скачивание с google play

если у кого так же- поделитесь как обошли.
